tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/AutoInstrumentation/AdoNet/MockDbCommands.cs

```diff
@@ -48,6 +54,103 @@ public override void Cancel()
 
         protected override DbDataReader ExecuteDbDataReader(CommandBehavior behavior) => null;
     }
+
+    public class MockDbParameter : DbParameter
+    {
+        public override DbType DbType { get; set; }
+
+        public override ParameterDirection Direction { get; set; } = ParameterDirection.Input;
+
+        public override bool IsNullable { get; set; }
+
+        public override string ParameterName { get; set; }
+
+        public override string SourceColumn { get; set; }
+
+        public override DataRowVersion SourceVersion { get; set; }
+
+        public override object Value { get; set; }
+
+        public override byte Precision { get; set; }
+
+        public override byte Scale { get; set; }
+
+        public override int Size { get; set; }
+
+        // TODO no clue what this is
```

Comment:
😂

Reply:
Yeah I need to come back through there and clean stuff up later